source/includes/authentication.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+require __DIR__ . '/../vendor/autoload.php';
+
+// start-scram-sha-256-client
+$uriOptions = [
```

Comment:
S: consider calling clientOptions or authOptions?

Reply:
Calling it uriOptions was a request from DBX, since the [client constructor](https://www.mongodb.com/docs/php-library/current/reference/method/MongoDBClient__construct/) takes two separate uriOptions and driverOptions parameters

Reply:
Aye, this is specific to PHP, since our constructors take two options arrays.

These are the argument names from the method signatures, which users might also see in their IDE and/or use with [named arguments](https://www.php.net/manual/en/functions.arguments.php#functions.named-arguments):

 * https://www.mongodb.com/docs/php-library/current/reference/method/MongoDBClient__construct/
 * https://www.php.net/manual/en/mongodb-driver-manager.construct.php